pinot-query-runtime/src/test/resources/queries/TableExpressions.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
High level question on table expression tests?

By table expressions do you mean CTEs ? CTEs create named queries using `WITH` afaik. 

May be we call these `Subqueries.json` ?

Reply:
table expression as defined in postgresql 7.2 section. (e.g. all clauses that alters the behavior of a SQL after choosing the tables) 
some of the sections we don't support is not included such as:
- [7.2.4. GROUPING SETS, CUBE, and ROLLUP](https://www.postgresql.org/docs/15/queries-table-expressions.html#QUERIES-GROUPING-SETS)
- [7.2.5. Window Function Processing](https://www.postgresql.org/docs/15/queries-table-expressions.html#QUERIES-WINDOW)